linux/cuda_info.bsh

```diff
@@ -651,11 +571,31 @@ function cuda_capabilities()
     #   #
     #   # strings nvcc | \grep '^sm_' | sed 's|sm_||' | tr '\n' ' '; echo
     #   # strings nvcc | \grep '^compute_' | sed 's|compute_||' | tr '\n' ' '; echo
+    #
+    # Nvidia decided to make things MORE confusing starting in 11 docs.
+    # https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html does
```

Comment:
I think you should mention the Table number here and in the next link

Reply:
I truly doubt the table numbers will stay the same, or will even use numbers. It'll probably be emojis or something else equally dumb